README.md

```diff
@@ -1,2 +1,54 @@
-# comp110-worksheet-B
+# comp110-worksheet-B (SCOTT CALVERT)
 Base repository for COMP110 worksheet B
+
+![pict](https://github.com/Mindengine76/comp110-worksheet-B/blob/master/WIN_20171019_19_32_18_Pro.jpg)
+
+First level of abstraction
+
+1)select random word
```

Comment:
No need to number lines, unless you are referring to them elsewhere (e.g. if you're expressing loops by saying things like "if XYZ then goto line 3" -- which is possible but not necessarily recommended)